### PR TITLE
Additional Translations

### PR DESCRIPTION
**Pull request by TurtIeSocks**
- Add forms
- Add types
- Add weather
- Everything else

## Description

Adds more translations to reduce the number of needed manual translations

## Motivation and Context

Saves everyone time and effort.

## How Has This Been Tested?

Tested locally and seems to work without issue.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by DarthStap** [comment]
I don't know how it is in other languages, but in Russian types have a terrible official translation (some of them are adjectives, some are adverbs, some are nouns and it really annoys me). Would it be possible to ovewrite them by translation from a locale file?

**Comment by jfberry** [comment]
This is absolutely as designed - lines in the poracle supplied translation override any of this 'systemised' translation

**Comment by TurtIeSocks** [comment]
@DarthStap if the official Russian translations are bad, please submit any overrides to my translations repo:
https://github.com/WatWowMap/pogo-translations
All translations in the manual folder override the official ones.

You could also submit it to Poracle but having them in my repo helps many poracle + many other pogo projects 😄

**Comment by jfberry** [comment]
So what do other translators think of these as a baseline? @ReuschelCGN @Petap0w 
I am happy to merge from a code point of view, but difficult for me to assess positive impact

**Comment by DarthStap** [comment]
> > I don't know how it is in other languages, but in Russian types have a terrible official translation (some of them are adjectives, some are adverbs, some are nouns and it really annoys me). Would it be possible to ovewrite them by translation from a locale file?
> 
> Is this something what all russian players want to? Or is it your personal opinion? :)

I haven't asked many people, but most of my game aquaintances agree with this.


> So what do other translators think of these as a baseline? @ReuschelCGN @Petap0w I am happy to merge from a code point of view, but difficult for me to assess positive impact

As long as I can overwrite things, this looks pretty solid.

**Comment by DarthStap** [comment]
> @DarthStap if the official Russian translations are bad, please submit any overrides to my translations repo: https://github.com/WatWowMap/pogo-translations All translations in the manual folder override the official ones.
> 
> You could also submit it to Poracle but having them in my repo helps poracle + many other pogo projects 😄

Will look into it when I have time

**Comment by ReuschelCGN** [comment]
@jfberry I will look / try it tonight in my poracle and will report back!

File: src/util/generateData.js [hunk 1]
```diff
@@ -4,11 +4,17 @@ const Fetch = require('node-fetch')
...
-})
+const fetch = async (url) => {
+	try {
+		const data = await Fetch(url)
+		if (!data.ok) {
+			throw new Error(`${data.status} ${data.statusText} URL: ${url}`)
+		}
+		return await data.json()
+	} catch (e) {
+		console.error(e, `Unable to fetch ${url}`)
```
Comment (fosJoddie) [inline src/util/generateData.js:15]: use `log.warn` instead of `console.error`

**Comment by TurtIeSocks** [comment]
Actually... there is an issue with weather in Poracle util not being capitalized. Went to change them and all the locales in the config folder but realized it's then conflicting with Castform forms. Thoughts @jfberry ?

**Comment by jfberry** [comment]
Yes, that is a fundamental with the way things work right now.  I think over time we should transition to classified lookups but it's quite a big job doing this and maintaining some compatibility

**Comment by TurtIeSocks** [comment]
Merge for now to reduce the need for one off translation PRs, then in a future PR we move everything over to an i18n like system?

**Pull request merged**